### ISSUE TITLE
custom usage inserts two newlines?

### ISSUE THREAD

**Issue by 0x5a4** [pacak/bpaf#22]
This snippet from `Info::render_help`
```
if let Some(u) = self.usage {
    write!(res, "{}\n\n", u)?;
} else {
    write!(res, "Usage: {}", parser_meta)?;
    // strip unnecessary spaces inserted by previous writes
    res.truncate(res.trim_end_matches(' ').len());
    write!(res, "\
}
```
inserts two newlines when a custom usage is defined, but only one if not? Is there a reason
for this? To me that seems like an oversight...

**Comment by pacak** [comment]
Thank you for the bugreport, you are totally correct, this is an oversight - I never actually used custom usage in my own code, will fix.

**Comment by pacak** [comment]
Released 0.4.10
